src/http/middleware/set-auth-headers.js

```diff
@@ -1,10 +1,22 @@
-
 // Sets auth headers if necessary
 
-function addAuthHeaders ({ bearerToken, headers, overrideHeaders=false }) {
-  if (overrideHeaders || !bearerToken) return
+function addAuthHeaders ({ 
+  auth,
+  bearerToken, 
+  headers, 
+  overrideHeaders=false, 
+}) {
+  if (overrideHeaders) return
+  let authToken = bearerToken
+  if (auth) {
+    const username = auth.username || ''
+    const password = auth.password || ''
+    authToken = window.btoa(`${ username }:${ password }`)
```

Comment:
Use an [isomorphic](https://www.npmjs.com/package/Base64) version of this to support node as well